experiments/gym/train_lunar_lander.py

```diff
@@ -163,25 +163,65 @@ def train_lunar_lander(args):
         ),
     ]
 
-    callbacks = logged_cbs + [
-        TensorboardLogger(
-            logged_cbs,
-            SummaryWriter(
-                log_dir=args.log_dir + "/" + args.name + "_{}".format(time.time())
+    if args.use_wandb:
+        import wandb
+
+        config = {
+            "wandb_project": args.name,
+            "wandb_run": args.wandb_run,
+            "hidden_dims": hidden_dims,
+            "batch_size": batch_size,
+            "learning_rate": learning_rate,
+            "rollout_len": rollout_len,
+        }
+
+        wandb_run = wandb.init(
+            project=config["wandb_project"],
+            name=config["wandb_run"],
+            config=wandb.helper.parse_config(
+                config, exclude=("wandb_project", "wandb_run")
             ),
-            100,
-        ),
-    ]
+        )
 
+        logger = [
+            WBLogger(
+                callbacks=logged_cbs,
+                log_interval=100,
+            ),
+        ]
+    else:
+        logger = [
```

Comment:
Nit: no need for a list here; you can just add it to the list after BackPropStepsTerminator instead. A bit less spacing.